### DISCUSSION TITLE
How to share AA4PP to my makers - i'm lost with all the access i need to give. What is the best approach ?

### DISCUSSION THREAD

**Discussion by tocri** [microsoft/powercat-creator-kit#341] [General]
Hi All,

After testing AA4PP and AA4PPA as an administrator for few days, i can approve that it is a very good product and simplify the deployment process.

Now i'm testing AA4PP as a maker and it's not clear what steps i need to follow in order to share the apps and permit maker to commit and deploy their solution from DEV to TEST/

i found this useful guide https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/guidance/alm-accelerator/setup-app-user-permissions.md but honestly it doesn't work. As a user i have plenty errors when i open AA4PP.

Could you help me ? 

Thanks a lot

**Comment by mikefactorial** [comment]
Hey @tocri looks like this was posted on the wrong repo. Saw a few more Issues opened from you over on https://github.com/microsoft/coe-starter-kit/issues. Feel free to post this as a question in that repo. Thanks
